Formula/groestlcoin.rb

```diff
@@ -0,0 +1,65 @@
+# typed: false
+# frozen_string_literal: true
+
+# Decentralized, peer to peer payment network
```

Comment:
where does this come from?
```suggestion
```

Reply:
If I remove:
```
# typed: false
# frozen_string_literal: true

# Decentralized, peer to peer payment network
```
the the last 2 checks only passes with `--skip-style` added otherwise it throws this error:
```
  * 1: col 1: No Sorbet sigil found in file. Try a `typed: false` to start (you can also use `rubocop -a` to automatically add this).
  * 1: col 1: Missing top-level documentation comment for `class Groestlcoin`.
  * 1: col 1: Missing frozen string literal comment.
```

Reply:
suggestion implemented